<launch>
  <rosparam file="$(find bt_mission_planner)/config/robosub21_auv4.yaml" />
  <node name="mission_planner" pkg="bt_mission_planner" type="mission_planner_node" respawn="true" output="screen"/>
</launch>